data/issues/emissions/emission_data_calculations.py

```diff
@@ -20,17 +20,17 @@
     'Mörbylånga': {
         2010: 248025000/1000, 2015: 255970000/1000, 2016: 239538000/1000,
         2017: 255783000/1000, 2018: 241897000/1000, 2019: 65176000/1000,
-        2020: 0, 2021: 0
+        2020: 0, 2021: 0, 2022: 0,
     },
     'Skövde': {
         2010: 356965000/1000, 2015: 358634000/1000, 2016: 384926000/1000,
         2017: 407633130/1000, 2018: 445630340/1000, 2019: 440504330/1000,
-        2020: 459092473/1000, 2021: 439174727/1000
+        2020: 459092473/1000, 2021: 439174727/1000, 2022: 406856000/1000,
```

Comment:
Future idea (maybe solve in a future issue some day): It would be nice to know where these numbers come from, and how they are calculated. For example, adding a link to the source or describing the method for updating these numbers close to the `CEMENT_DEDUCTION` constant.

Reply:
Good point! The sources were present in another part of the file, moved them to make it clearer where the data is sourced from.